08-verification-right.md

```diff
@@ -60,7 +60,7 @@ Run the code by running this command in your terminal, still within your `kilt-r
 node nonce.js
 ```
 
-You should see in your logs the `uuid`, that will be used as a nonce; it should look like something like this: `9dbbad48-aad0-4280-aeca-4c4072c82625`.
+You should see in your logs the UUID encoded as an HEX string, that will be used as a nonce; it should look like something like this: `0x942ab89b01671faeec84a76f4a8eae9b57ec12bf06157f8a87315cd29a5e0d25`.
```

Comment:
I wasn't aware we are hashing the uuid in our sdk. Is it still a uuid in that case?

Reply:
A comment above states that
> (A UUID is not _strictly_ a nonce, because it's not a number, but here we'll refer to it as nonce).

while the hash we are using strictly speaking *is* a number.